cirq-core/cirq/experiments/xeb_fitting.py

```diff
@@ -413,13 +413,15 @@ def _mean_infidelity(angles):
         method='nelder-mead',
     )
 
-    final_params = dict(zip(names, optimization_result.x))
+    final_params: Dict[Union[str, sympy.Expr], Union[float, sympy.Expr]] = dict(
+        zip(names, optimization_result.x)
+    )
     fidelities_df = benchmark_2q_xeb_fidelities(
         sampled_df, parameterized_circuits, cycle_depths, param_resolver=final_params
     )
     return XEBCharacterizationResult(
         optimization_results={pair: optimization_result},
-        final_params={pair: final_params},
+        final_params={pair: final_params},  # type: ignore
```

Comment:
I'd suggest narrowing the type ignore to the specific error code listed by mypy, e.g.
```suggestion
        final_params={pair: final_params},  # type: ignore[dict-item]
```
This can be a hint about what would need to be fixed to remove the type ignore comment.